examples/node/package.json

```
@@ -7,5 +7,10 @@
     "test": "tsc -p tsconfig.json && node node.js"
   },
   "author": "@gamebridgeai",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "@gamebridgeai/ts_serialize": "0.2.4",
```

```suggestion
    "@gamebridgeai/ts_serialize": "*",
```

I think this always grabs the latest so we don't need to manually update for releases, maybe we add it for all our packages here.

Thoughts.

---

Sounds good. We currently don't have a way to automatically update the lockfile, but TBH I don't think it's a big deal b/c we don't need to update the dependency for `ts_serialize`.

The lockfile it's self is ignored unless explicitly added so we shouldn't have any issue with people accidentally committing an updated lockfile. The tests links the build release, so even if the lockfile is pointing to an aincent version of ts_serialize we shouldn't have any problems.